### PR TITLE
Initial commit of the SFC application.

### PR DESCRIPTION
**Pull request by grahamc**
You can see the anwers-only diff here: https://github.com/NixOS/nixos-foundation/compare/8f9ea...sfc-app

cc #3 @zimbatm @edolstra @rbvermaa @copumpkin @armijnhemel 

TODOs are marked with `!!!`, if you have information to add, please leave a comment, or send a PR to this branch -- don't push to this branch directly please.

### CODE REVIEW

File: software-freedom-conservancy/application.txt [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+*** Approximately how many users does your project have, and what items
+    lead you to believe your userbase is of a particular size (e.g., post
+    counts to your user mailing list)?
+
+We merge about 1,000 pull requests each month, from about 300
+different people. We have about 2,500 stars and about 2,800 forks on
+GitHub. The main IRC channel stays above 800 people on a consistent
+basis.
+
+Some traffic stats from late 2018:
```
Comment (edolstra) [inline software-freedom-conservancy/application.txt]: Late 2017.

File: software-freedom-conservancy/application.txt [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+
+We have talked with NLNet about being homed under them. They would be
+happy to do so, however their model makes it challenging to receive
+regular and smaller contributions.
+
+We also have run our own foundation, the NixOS Foundation, however it
+is not not-for-profit, and transparency reporting is not a nice way to
+spend contributor time.
+
+!!! Consider SPI and/or LF
```
Comment (zimbatm) [inline software-freedom-conservancy/application.txt]: Just comparing the projects that each foundation hosts:

[SPI](https://www.spi-inc.org/projects/) hosts a number of other distros like Debian, Arch Linux and OpenWRT.

[Linux Foundation](https://www.linuxfoundation.org/projects/) seems to be more corportate and cloud-focused. Aside from the Linux Kernel they also host Kubernetes and a bunch of related projects. I think Yocto is the only Linux distribution in there.

[SFC](https://sfconservancy.org/projects/current/) has more infrastructure projects like busybox, Git, Mercurial. It also hosts the Homebrew package manager.

Providing that other distributions might have the same needs as us and they probably did their own research, SPI seems like the best fit.

File: software-freedom-conservancy/application.txt [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+ - Packet.net provides several build servers for NixOS, in exchange
+   for us creating, maintaining, and providing installation images for
+   their service.
+ - TUDelft hosts a few build servers for us, and in exchange has a
+   logo on our website (same as above.)
+ - Universiteit Utrtech does ... WHAT? ...  and in exchange has
+   a logo on our website (same as above.)
+ - !!! WHAT IS 3TU / 4TU?
+ - !!! LIST OF FUNDING GRANTS NIX HAS RECEIVED?
+ - !!! WHAT DOES SNABB DO FOR US?
```
Comment (zimbatm) [inline software-freedom-conservancy/application.txt]: I think Snabb was just funding Domen's work for a while

File: software-freedom-conservancy/application.txt [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+ - !!! MacStadium provides Macs, but I'm unclear on the relationship.
```
Comment (zimbatm) [inline software-freedom-conservancy/application.txt]: I contacted MacStadium independently to get some funding (I didn't know about the existing relationship) and the plan was to provide some machines in exchange of a logo on the website.

File: software-freedom-conservancy/application.txt [hunk 5]
```diff
@@ -0,0 +1,413 @@
...
+    contributors.  If you project has contributors who have been inactive
+    for more than five years, you need only to list such inactive
+    contributors if they made substantial contributions.
+
+Please clone the nixpkgs repository and run git shortlog -s -n -e.
+
+Key individuals:
+
+ - Eelco Dolstra, edolstra@gmail.com, Tweag.io. Previously: LogicBlox
+ - !!!
```
Comment (zimbatm) [inline software-freedom-conservancy/application.txt]: Should probably add the other board members here

File: software-freedom-conservancy/application.txt (hunk 2 above)
Comment (davidak) [reply software-freedom-conservancy/application.txt]: >Providing that other distributions might have the same needs as us and they probably did their own research, SPI seems like the best fit.

That's speculation. Maybe we can just ask them?

File: software-freedom-conservancy/application.txt (hunk 4 above)
Comment (davidak) [reply software-freedom-conservancy/application.txt]: I initially contacted MacStadium for sponsoring, later @rbvermaa was in contact with them. He will be able to provide all needed details.

**Comment by zimbatm** [comment]
closing. see #1

**Pull request closed without merging**